be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -166,8 +166,58 @@ struct ProcessHashTableProbe {
               _items_counts(join_node->_items_counts),
               _build_block_offsets(join_node->_build_block_offsets),
               _build_block_rows(join_node->_build_block_rows),
-              _rows_returned_counter(join_node->_rows_returned_counter) {}
+              _rows_returned_counter(join_node->_rows_returned_counter),
+              _search_hashtable_timer(join_node->_search_hashtable_timer),
+              _build_side_output_timer(join_node->_build_side_output_timer),
+              _probe_side_output_timer(join_node->_probe_side_output_timer) {}
+
+    // output build side result column
+    Status build_side_output_column(MutableColumns& mcol, int column_offset, int column_length, int size) {
+        constexpr auto is_semi_anti_join = JoinOpType::value == TJoinOp::RIGHT_ANTI_JOIN ||
```

Comment:
The Function only return OK ？ seems not need to return Status.

Reply:
i agree.